compiler/apps/playground/components/Editor/EditorImpl.tsx

```diff
@@ -59,20 +70,25 @@ function parseInput(input: string, language: 'flow' | 'typescript'): any {
     });
   }
 }
-
+function checkExplicitMemoizeDirectives(
+  directives: Array<t.Directive>,
+): MemoizeDirectiveState {
+  if (findDirectiveEnablingMemoization(directives).length) {
+    return MemoizeDirectiveState.Enabled;
+  }
+  if (findDirectiveDisablingMemoization(directives).length) {
+    return MemoizeDirectiveState.Disabled;
+  }
+  return MemoizeDirectiveState.Undefined;
+}
 function parseFunctions(
   source: string,
   language: 'flow' | 'typescript',
-): Array<
-  | NodePath<t.FunctionDeclaration>
-  | NodePath<t.ArrowFunctionExpression>
-  | NodePath<t.FunctionExpression>
-> {
-  const items: Array<
-    | NodePath<t.FunctionDeclaration>
-    | NodePath<t.ArrowFunctionExpression>
-    | NodePath<t.FunctionExpression>
-  > = [];
+): Array<{
+  compile: boolean;
```

Comment:
I think something like `compilationEnabled/Disabled` might be more accurate?

Reply:
done